### PR TITLE
docs, probes: fix readiness and liveness examples

### PR DESCRIPTION
**Pull request by jcanocan**
Yaml files of readiness and liveness examples have been updated to work with current kubevirt versions. Also, containerdisks images have been updated.

### CODE REVIEW

File: docs/virtual_machines/liveness_and_readiness_probes.md [hunk 1]
```diff
@@ -28,47 +28,54 @@ VirtualMachineInstance, after an initial delay of 120 seconds. The
...
+    initialDelaySeconds: 120
+    periodSeconds: 20
+    httpGet:
+      port: 1500
+    timeoutSeconds: 10
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    containerDisk:
+      image: quay.io/containerdisks/fedora
```
Comment (0xFelix) [inline docs/virtual_machines/liveness_and_readiness_probes.md:67]: An explicit tag for the image is missing here. I'd suggest `latest`.

File: docs/virtual_machines/liveness_and_readiness_probes.md [hunk 2]
```diff
@@ -77,95 +84,111 @@ VirtualMachineInstance, after an initial delay of 120 seconds. The
...
+    initialDelaySeconds: 120
+    periodSeconds: 20
+    tcpSocket:
+      port: 1500
+    timeoutSeconds: 10
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    containerDisk:
+      image: quay.io/containerdisks/fedora
```
Comment (0xFelix) [inline docs/virtual_machines/liveness_and_readiness_probes.md:123]: Same as above.

File: docs/virtual_machines/liveness_and_readiness_probes.md [hunk 3]
```diff
@@ -77,95 +84,111 @@ VirtualMachineInstance, after an initial delay of 120 seconds. The
...
+    timeoutSeconds: 10
+    failureThreshold: 3
+    successThreshold: 3
+    httpGet:
+      port: 1500
+  terminationGracePeriodSeconds: 0
+  volumes:
+  - name: containerdisk
+    containerDisk:
+      image: quay.io/containerdisks/fedora
```
Comment (0xFelix) [inline docs/virtual_machines/liveness_and_readiness_probes.md:180]: Same as above.

**Review by 0xFelix — COMMENTED** [review]
Looks good except for image tags missing. I like the removal of disabling SELinux! :+1: Can we use another containerdisk in the watchdog example too?

File: docs/virtual_machines/liveness_and_readiness_probes.md (hunk 3 above)
Comment (jcanocan) [reply docs/virtual_machines/liveness_and_readiness_probes.md]: All done! Thanks for your suggestions :)

File: docs/virtual_machines/liveness_and_readiness_probes.md [hunk 4]
```diff
@@ -200,13 +223,12 @@ Operating System is focused on. One can configure the `i6300esb` watchdog
...
 
 ```yaml
----
 apiVersion: kubevirt.io/v1
 kind: VirtualMachineInstance
 metadata:
   labels:
-    special: vmi-with-watchdog
-  name: vmi-with-watchdog
+    special: vmi-with-watchdog-new
```
Comment (0xFelix) [inline docs/virtual_machines/liveness_and_readiness_probes.md:230]: Do we need the suffix `-new`?

Comment (jcanocan) [reply docs/virtual_machines/liveness_and_readiness_probes.md]: No. My bad. I forgot to remove it after testing with the old and the new version of the yaml. Fixed!

**Comment by jcanocan** [comment]
@cwilkers  Could you please take a look at this? Thanks!

**Comment by cwilkers** [comment]
Looks good to me,

/approve

**Comment by jcanocan** [comment]
Many thanks! @cwilkers :)

**Pull request merged**